src/types/argument.js

```diff
@@ -24,15 +24,19 @@ export function initArgument(type) {
       g.base_info.unref(info);
       return result;
     }
+    default: {
+      return cast_ptr_u64(cast_buf_ptr(new Uint8Array(1)));
```

Comment:
Actually, I believe all arguments should be initialised to an empty pointer (`GIArgInfo`)